### DISCUSSION TITLE
Do you have any plans to add a serverless framework?

### DISCUSSION THREAD

**Discussion by changhyuni** [Checkmarx/kics#4916] [Q&A]
Hi.
Our company is using KICS well. With the introduction of serverless framework, I hope that serverless framework yaml will also be supported by kics.

**Comment by kaplanlior** [comment]
Hi @changhyuni 

Thanks for the question - just to very your refer to https://github.com/serverless/serverless or did you mean something else ?

Would your company be interested with helping support this framework?

**Comment by changhyuni** [comment]
Yes, that’s right!

**Reply by Mikisharon** [reply]
Hi @changhyuni,
We will research that serverless framework in one of the coming Sprints and update on outcome if/when we are going to support that.

**Reply by lior-orca** [reply]
@Mikisharon  
is there an answer on when Kics aim to support it? will it be part of Kics 1.6 content?

**Comment by changhyuni** [comment]
@Mikisharon Thanks you!!

**Reply by cx-nuno-oliveira** [reply]
Hi @lior-orca  and @changhyuni. 
Yes, we will be supporting Serverless Framework in KICS 1.6.
In a few days we will be disclosing a preliminary date for 1.6 release. 
Stay tuned ;)
